src/components/Dialog/UpdateNotificationDialog/Container.vue

```diff
@@ -73,12 +73,13 @@ const handleSkipThisVersionClick = (version: string) => {
 };
 
 // ダイアログを開くかどうか
-watchEffect(() => {
+const stopWatchEffect = watchEffect(() => {
   if (
     props.canOpenDialog &&
     newUpdateResult.value.status == "updateAvailable"
   ) {
     isDialogOpenComputed.value = true;
+    stopWatchEffect();
```

Comment:
isDialogOpenComputed.valueがfalseになる度にここでtrueが再代入され、ダイアログが閉じられないというバグの修正のために一回開いたらあとはwatchEffectを止める処理を追加しました。

Reply:
あっほんとですね。。

修正方法良さそうに思いました！！
意図コメント追加しておくと分かりやすそう！（この場合、やってることから意図を読み取れないので）
```suggestion
    stopWatchEffect(); // ダイアログを再表示させない
```